src/proxy-ui-api/frontend/src/views/AppError.vue

```diff
@@ -23,12 +23,115 @@
    OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
    THE SOFTWARE.
  -->
+
 <template>
-  <v-layout align-center justify-center column>oops, page not found</v-layout>
+  <div class="xrd-view-common">
+    <TabsBase />
+    <AlertsContainer />
+    <v-container>
+      <div class="xrd-view-title pt-6">{{ $t('404') }}</div>
+      <v-layout align-center justify-center column fill-height elevation-0>
+        <v-card flat class="xrd-card custom-card">
+          <v-card-title> </v-card-title>
+
+          <v-card-text>
+            <div class="content-wrap">
+              <v-img
```

Comment:
The image does not scale down for smaller screens.

Reply:
Fixed